.github/workflows/build-images.yml

```
@@ -0,0 +1,76 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # packages: write
+
+jobs:
+  build-container-image:
```

```suggestion
  build-container-image:
    if: github.repository_owner == 'FreshRSS'
```

https://github.com/orgs/community/discussions/26409#discussioncomment-3251822

Maybe that could be done smarter. It would be OK to let the build occur, as long as there is no push attempt

---

I add some logic to prevent the push on non upstream repos